viewer/packages/tools/src/Geomap/Geomap.ts

```diff
@@ -10,6 +10,7 @@ import { LatLongPosition, MapConfig, MapProviders } from './MapConfig';
 export class Geomap {
   private readonly _viewer: Cognite3DViewer;
   private _map: GEOTHREE.MapView;
+  private _intervalId: any = 0;
```

Comment:
Tip: instead of using any you can use `ReturnType<typeof setInterval>`.

Reply:
We can use to equal/assign `ReturnType<typeof setInterval>` type, I used the same but could not get through it to access the value as number, so made it as `any`.